provisioning/tasks/drupal.yml

```diff
@@ -13,14 +13,14 @@
 
 - name: Generate Drupal site with drush makefile.
   command: >
-    drush make -y {{ drush_makefile_path }} --no-gitinfofile
+    {{ drush_path }} make -y {{ drush_makefile_path }} --no-gitinfofile
```

Comment:
Is this a new variable? I'm fuzzy on it, but if it's definitely defined in all circumstances, I'm okay with using the var here. Otherwise, this seems unrelated to the overall PR.